common-tools/clas-analysis/src/main/java/org/jlab/analysis/eventmerger/ADCTDCMerger.java

```diff
@@ -59,17 +59,23 @@ public ADCTDCMerger(String[] dets, boolean dhits) {
             int adc         = bankDGTZ.getInt("ADC", i);
             float time      = bankDGTZ.getFloat("time", i);
             short ped       = bankDGTZ.getShort("ped", i);
-            if(adc<=0) continue;
-            if(detector == DetectorType.BST.getName()) {
+            
+            if(adc<=0) 
+                if(!detector.equals(DetectorType.BST.getName()) && !detector.equals(DetectorType.BMT.getName())) 
```

Comment:
Should cross check whether adc<0 hits should be used based on the FE behaviors of BST and BMT

Reply:
Issue #783